Modules/OptionHolder.cs

```diff
@@ -1575,6 +1576,17 @@ public static void Load()
 
         Wraith.SetupCustomOption();
 
+        TextOptionItem.Create(10000015, "RoleType.NeutralApocalypse", TabGroup.NeutralRoles)
+            .SetGameMode(CustomGameMode.Standard)
+            .SetColor(new Color32(127, 140, 141, byte.MaxValue));
+
+        Baker.SetupCustomOption();
+
+        Berserker.SetupCustomOption();
+
+        PlagueBearer.SetupCustomOption();
+
+        SoulCollector.SetupCustomOption();
```

Comment:
remember to make a new thing "public override Custom_RoleType ThisRoleType => Custom_RoleType.Apocalypse; in the respective roles and automate it here.

Reply:
when in the world did that exist 😭
I'll update when I fix everything